### PR TITLE
Refactored rescan_appointments into search_appointments eliminating a…

### PR DESCRIPTION
**Pull request by lusor**
… duplicate click

### CODE REVIEW

**Comment by alfonsrv** [comment]
Hey Volker,

good one - also already thought about it. I removed the `except Exception` and added a logging event, so we don't just `pass` to allow for better debugging in the future.

**Comment by alfonsrv** [comment]
Merging to main. Thanks for contributing.

File: impf/browser.py [hunk 1]
```diff
@@ -322,14 +322,23 @@ def search_appointments(self) -> bool:
         title = self.wait.until(EC.presence_of_element_located((By.XPATH, '//h1')))
         assert title.text == 'Onlinebuchung für Ihre Corona-Schutzimpfung'
 
+        try:
+            close = self.driver.find_elements_by_xpath('//button[contains(text(), "Abbrechen")]')[-1]
+            close.click()
+            sleep(1)
+        except NoSuchElementException:
```
Comment (lusor) [inline impf/browser.py:329]: This won't be a NoSuchElementException, but a "NotClickableException" (I forgot the corret name).

Also, the first time `search_appointments` is executed, we expect the exception to be raised as the popup hasn't been opened yet. So maybe this should be of log level DEBUG?

Comment (alfonsrv) [reply impf/browser.py:329]: You're right; depends on whether or not the popup window is already loaded. Pushed a quick commit.

**Pull request merged**